docs/multicluster/user-guide.md

```diff
@@ -486,6 +487,20 @@ exported a Service: `default/nginx` with TCP Port `80`, other clusters can only
 export the same Service with the same Ports definition including Port names. At
 the moment, Antrea Multi-cluster supports only IPv4 multi-cluster Services.
 
+By default, a multi-cluster Service will use the exported Services' ClusterIPs (the
+original Service ClusterIPs in the export clusters) as the endpoints. Since
```

Comment:
```suggestion
original Service ClusterIPs in the export clusters) as Endpoints. Since
```

Reply:
Humm..

Let me merge this suggestion, as `Endpoints` can mean the Endpoints resource here.